lib/jekyll-webp/defaults.rb

```diff
@@ -6,8 +6,15 @@ module Webp
     DEFAULT = {
       'enabled'   => false,
 
-      # The quality of the webp conversion 0 to 100 (where 100 is least lossy)
-      'quality'   => 75,
+      # The flags to pass to the webp binary. For a list of valid parameters check here:
+      # https://developers.google.com/speed/webp/docs/cwebp#options
+      'flags'     => "-q 75",
```

Comment:
I agree partly that we don't want to map the entire command line of the webp tool, however I'd like the use of this gem to be simple for a novice user so abstracting some of the flags out to independant variables makes sense. With that said, I'd like to keep the quality parameter separate from others as this is probably the one that most users would rely on. We should however keep the `flags` one as well and simply append it.